src/components/controls/controls.tsx

```
@@ -1,8 +1,9 @@
 import { observer } from "mobx-react-lite";
 
-import { IControls, ISliderControl, IArrayControl } from "../../types/control";
+import { IControls, ISliderControl, IArrayControl, ISetControl } from "../../types/control";
 import Slider from "./slider";
 import Array from "./array";
```

Biome (2.1.2)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)